app/assets/javascripts/select-kit/addon/components/combo-box/combo-box-header.js

```diff
@@ -11,12 +11,13 @@ export default SingleSelectHeaderComponent.extend({
   caretDownIcon: reads("selectKit.options.caretDownIcon"),
   shouldDisplayClearableButton: and("clearable", "value"),
 
-  caretIcon: computed(
-    "selectKit.isExpanded",
-    "caretUpIcon",
-    "caretDownIcon",
-    function () {
+  caretIcon: computed("selectKit.isExpanded", "caretUpIcon", "caretDownIcon", {
+    get() {
       return this.selectKit.isExpanded ? this.caretUpIcon : this.caretDownIcon;
-    }
-  ),
+    },
+
+    set(key, value) {
+      return (this._caretIcon = value);
```

Comment:
If we're not going to use the value it's probably better to fix the issue in offending plugins 😃